### PR TITLE
Adding documentation to the function determineElectrodeType.

### PR DESCRIPTION
**Pull request by Daniel-Tojal**

### CODE REVIEW

File: src/Functions/determineElectrodeType.m [hunk 1]
```diff
@@ -1,12 +1,25 @@
...
-% 2016  - 2017
-% mail@andreashusch.de
+% INPUT: 
+%    peakDistances:     The peak distances between the signals, provided by the inserted 
+%                       data sheet.   
+%
+% OUTPUTS: 
+%    electStruct:       Electrode Type.
+%    rms:               Root-mean-square of the distance between the electrode signals.
+%    d:                 ?
```
Comment (laurentheirendt) [inline src/Functions/determineElectrodeType.m]: @adhusch, we need your input here...

Comment (Daniel-Tojal) [reply src/Functions/determineElectrodeType.m]: I will look into this, i didn't notice i left the question mark in it until after we had our meeting

Comment (adhusch) [reply src/Functions/determineElectrodeType.m]: d: euclidian distance to the closest electrode prototype

**Comment by laurentheirendt** [comment]
Thanks @Daniel-Tojal 👍

**Pull request merged**